open-metadata-implementation/access-services/digital-service/digital-service-spring/src/main/java/org/odpi/openmetadata/accessservices/digitalservice/spring/DigitalServiceResource.java

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.digitalservice.spring;
+
+import org.odpi.openmetadata.accessservices.digitalservice.server.DigitalServiceRESTServices;
+import org.odpi.openmetadata.accessservices.digitalservice.rest.DigitalServiceRequestBody;
+import org.odpi.openmetadata.commonservices.ffdc.rest.GUIDResponse;
+import org.springframework.web.bind.annotation.*;
```

Comment:
wildcard

Reply:
added

Reply:
Hi - I pulled the latest code to check, and there's no change there, so perhaps the update didn't get in? Not a massive issue, but worth checking in case any changes you intended are omitted?